src/utils/validators.test.js

```diff
@@ -0,0 +1,69 @@
+// ------------------------------------------------------------------------------ //
+// ------------------------------ UNIT TESTS ------------------------------------ //
+// ------------------------------------------------------------------------------ //
+
+// Enter "npm test" to run tests.
+
+import { textFieldValidator } from "./validators";
+
+test("Empty textField", () => {
+  const value = textFieldValidator("");
+  expect(value).toBeTruthy();
+});
+
+function getRandomText(length) {
```

Comment:
There is an excellent library that helps to create random content using in tests:
`https://github.com/marak/Faker.js/` 

We don't need to reinvent the wheel ;)